### PR TITLE
Add conceal enabled config

### PR DESCRIPTION
**Pull request by t1gu1**
- The conceal use the value define in the config
- The conceal will be enabled by default

P.S. The lsp.color.request is call externally of the conceal.lua file to not create a circular dependency using the conceal inside the lsp file.

### CODE REVIEW

**Comment by luckasRanarison** [comment]
I think a simpler solution would be adding BufEnter autocmd in the initial.lua and add enable_conceal as callback depending on the config.

**Comment by t1gu1** [comment]
So true.
I change it as you said.

**Comment by luckasRanarison** [comment]
Thanks for the PR!

File: lua/tailwind-tools/init.lua [hunk 1]
```diff
@@ -36,14 +37,14 @@ M.setup = function(options)
...
-    callback = function()
-      -- if not state.color.enabled then return end
-      if config.options.conceal.enabled then conceal.enable() end
-    end,
-  })
+  if config.options.conceal.enabled then
+    vim.api.nvim_create_autocmd("BufEnter", {
+      group = vim.g.tailwind_tools.conceal_au,
+      callback = function()
+        if not state.conceal.enabled then conceal.enable() end
```
Comment (t1gu1) [inline lua/tailwind-tools/init.lua:44]: Nothings set the state to true here.
So it will not works.

**Pull request merged**